scheduler/ruff.toml

```diff
@@ -0,0 +1 @@
+exclude = ["scheduler/_vendor/"]
```

Comment:
We have to set a standard for this for all our mcaf "python" projects but for now, do you agree with adding this?

This is extracted from other projects like awsfindingsmanagerlib and other python projects I found.

```
exclude = [
"scheduler/_vendor/"
]

line-length = 121
indent-width = 4

# Allow fix for all enabled rules (when `--fix`) is provided.
fixable = ["ALL"]

[format]
quote-style = "single"
indent-style = "space"
line-ending = "auto"
```